apps/odyssey-stats/package.json

```diff
@@ -29,6 +29,7 @@
 		"@automattic/calypso-polyfills": "workspace:^",
 		"@automattic/calypso-url": "workspace:^",
 		"@automattic/components": "workspace:^",
+		"@wordpress/icons": "workspace:^",
```

Comment:
How did you add this dependency? It looks like the yarn lockfile wasn't updated and is causing build failures. I recommend using `yarn add some-dependency/here` to avoid this.